test/functional/rpc_psbt.py

```diff
@@ -406,6 +408,39 @@ def test_psbt_named_parameter_handling(self):
 
         self.log.info("PSBT parameter handling test completed successfully")
 
+    def test_psbt_with_invalid_signature(self):
+        self.log.info("Test descriptorprocesspsbt with invalid signature in signed PSBT")
+
+        def_wallet = self.nodes[0].get_wallet_rpc(self.default_wallet_name)
+        outputs = [{def_wallet.getnewaddress(address_type="bech32m"): 1}]
+        def_wallet.send(outputs)
+        self.generate(self.nodes[0], 6)
+
+        utxos = [utxo for utxo in def_wallet.listunspent() if utxo["desc"].startswith("tr")]
+        unsigned_psbt = def_wallet.walletcreatefundedpsbt(utxos, [{def_wallet.getnewaddress(): 0.5}])["psbt"]
+        descs = [desc for desc in def_wallet.listdescriptors(True)["descriptors"] if desc["desc"].startswith("tr")]
+
+        # unload wallet so as to not use the wallet RPC in further process PSBT calls
+        def_wallet.unloadwallet()
+        
```

Comment:
In 7e468a6fb01b36edb6955ccb2b869ec1063a891c "rpc: Fix descriptorprocesspsbt internal bug on invalid signatures"

Extra whitespace on this line